src/library/Uno.Material/Styles/Controls/v2/RadioButton.xaml

```diff
@@ -1,55 +1,55 @@
-
-            <!--#region Fill-->
-            <StaticResource x:Key="RadioButtonOuterEllipseFill" ResourceKey="SystemControlTransparentBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseFillPointerOver" ResourceKey="SystemControlTransparentBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseFillPressed" ResourceKey="SystemControlTransparentBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseFillDisabled" ResourceKey="SystemControlTransparentBrush" />
-
-            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFill" ResourceKey="PrimaryBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFillPointerOver" ResourceKey="PrimaryBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFillPressed" ResourceKey="PrimaryBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseCheckedFillDisabled" ResourceKey="PrimaryBrush" />
-            <!--#endregion-->
-
-            <!--#region HoverRing-->
-            <StaticResource x:Key="MaterialRadioButtonHoverRingFillPointerOver" ResourceKey="PrimaryHoverBrush" />
-            <StaticResource x:Key="MaterialRadioButtonHoverRingFillPressed" ResourceKey="PrimaryFocusedBrush" />
-            <!--#endregion-->
-
-            <!--#region Typo-->
-            <StaticResource x:Key="RadioButtonFontFamily" ResourceKey="MaterialRegularFontFamily" />
-            <StaticResource x:Key="RadioButtonFontWeight" ResourceKey="BodySmallFontWeight" />
-            <StaticResource x:Key="RadioButtonFontSize" ResourceKey="BodySmallFontSize" />
-            <StaticResource x:Key="RadioButtonCharacterSpacing" ResourceKey="BodySmallCharacterSpacing" />
-            <!--#endregion-->
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:android="http://uno.ui/android"
+					xmlns:ios="http://uno.ui/ios"
+					xmlns:wasm="http://uno.ui/wasm"
+					mc:Ignorable="ios android wasm">
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<!--#region Foreground-->
+			<StaticResource x:Key="RadioButtonForeground" ResourceKey="OnSurfaceMediumBrush" />
+			<StaticResource x:Key="RadioButtonForegroundPointerOver" ResourceKey="OnSurfaceMediumBrush" />
+			<StaticResource x:Key="RadioButtonForegroundPressed" ResourceKey="OnSurfaceMediumBrush" />
+			<StaticResource x:Key="RadioButtonForegroundDisabled" ResourceKey="OnSurfaceLowBrush" />
+			<!--#endregion-->
+
+			<!--#region Stroke-->
+			<StaticResource x:Key="RadioButtonOuterEllipseStroke" ResourceKey="OnBackgroundMediumBrush" />
+			<StaticResource x:Key="RadioButtonOuterEllipseStrokePointerOver" ResourceKey="OnBackgroundMediumBrush" />
+			<StaticResource x:Key="RadioButtonOuterEllipseStrokePressed" ResourceKey="OnBackgroundMediumBrush" />
+			<StaticResource x:Key="RadioButtonOuterEllipseStrokeDisabled" ResourceKey="OnBackgroundDisabledBrush" />
```

Comment:
@Marc-Antoine-Soucy I think the ellipses colors needs an update also for Disable State please
![image](https://github.com/unoplatform/Uno.Themes/assets/16295702/8177a8fb-9711-4430-b315-11e2f2f7bbb2)